dotnet/src/Connectors/Connectors.OpenAI/AzureSdk/OpenAIChatMessageContent.cs

```diff
@@ -82,7 +80,7 @@ public IReadOnlyList<OpenAIFunctionToolCall> GetOpenAIFunctionToolCalls()
     }
 
     private static IReadOnlyDictionary<string, object?>? CreateMetadataDictionary(
-        IReadOnlyList<ChatCompletionsToolCall> toolCalls,
+        IReadOnlyList<ChatCompletionsFunctionToolCall> toolCalls,
```

Comment:
I would have thought just changing the type stored into the metadata dictionary word be sufficient. Is it not?

Reply:
I'm trying only that and see if it suffices.

Reply:
If it doesn't, we'll need to do something else... I don't think we can change the public surface area like this.